### PR TITLE
*.farmanager.com

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

File: src/chrome/content/rules/FarManager.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="FarManager">
+    <target host="farmanager.com" />
+    <target host="www.farmanager.com" />
+    <target host="forum.farmanager.com" />
+    <target host="plugring.farmanager.com" />
+    <target host="api.farmanager.com" />
+    <target host="bugs.farmanager.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/FarManager.xml]: Please also add `enforum.farmanager.com` and sort the targets alphabetically (you can keep `^farmanager.com` and `www.farmanager.com` on top).

Comment (techtonik) [inline src/chrome/content/rules/FarManager.xml]: Done.

File: src/chrome/content/rules/FarManager.xml [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <ruleset name="FarManager">
-    <target host="farmanager.com" />
+    <target host="^farmanager.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/FarManager.xml]: You don't need to write the `^`, sorry.

Comment (techtonik) [inline src/chrome/content/rules/FarManager.xml]: No problem.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**